.github/workflows/main.yml

```
@@ -53,3 +53,24 @@ jobs:
         env:
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
 
+      - name: DevDox Locust Test Generator
+        working-directory: devdox
```

actionlint (1.7.8)

"working-directory" is not available with "uses". it is only available with "run"

(syntax-check)